scala/tempo/src/main/scala/com/databrickslabs/tempo/TSDF.scala

```diff
@@ -271,10 +276,30 @@ private[tempo] sealed class BaseTSDF(val df: DataFrame,
 	def windowOverRange(length: Long, offset: Long): WindowSpec =
 		windowBetweenRange(offset, (offset + length - 1))
 
-	def asofJoin(rightTSDF : TSDF) : TSDF = {
-      rightTSDF
+	// asof Join functions
```

Comment:
I'm using a pretty hacky if/else statement to get different versions of asOfJoin. I'd have preferred something like overloaded methods, but I couldn't get it to work. May try it again in another commit. Happy to listen to any suggestions!